internal/pkg/block/block.go

```diff
@@ -33,9 +33,6 @@ type Block struct {
 	// ElectionProof is the vrf proof giving this block's miner authoring rights
 	ElectionProof crypto.VRFPi
 
-	// EPoStInfo wraps all data for verifying this block's Election PoSt
```

Comment:
I would avoid block header changes to avoid master rebase issues and to prevent conflicts for the changes coming in #4005